frontend/app/src/components/table-filter/FilterDropdown.vue

```diff
@@ -136,11 +144,47 @@ watch(selectedSuggestion, async () => {
 });
 
 const highlightedTextClasses = 'text-subtitle-2 text-rui-text-secondary';
+
+function handleEnter() {
```

Comment:
what is this `handleEnter` does?

Reply:
I'm not sure where is this called, I put console.log here, it never gets called, by pressing enter while any button is highlighted